### PR TITLE
vdk-control-cli: Support 3.10

### PR DESCRIPTION
**Pull request by gageorgiev**
This change enables support for Python 3.10.
Addresses #390 

Signed-off-by: gageorgiev <gageorgiev@vmware.com>

### CODE REVIEW

File: projects/vdk-control-cli/cicd/build.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ pip install -e .
 
 echo "Run unit tests and generate coverage report"
 pip install pytest-cov
+pip install pytest==6.2.4 # TODO: there is some regression with the newest version, remove this when possible
```
Comment (tozka) [inline projects/vdk-control-cli/cicd/build.sh:33]: What is the regression? Can you link an issue/ticket to pytest  
Do we need to open an internal github issue to address this?

File: projects/vdk-control-cli/.gitlab-ci.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ image: "python:3.9"
   stage: build
   before_script:
     - cd projects/vdk-control-cli
+    - pip install --upgrade pytest pip # TODO: REMOVE THIS AT SOME POINT
```
Comment (tozka) [inline projects/vdk-control-cli/.gitlab-ci.yml:13]: TODO? Is this necessary? as it's installed by build.sh ?

Comment (gageorgiev) [reply projects/vdk-control-cli/.gitlab-ci.yml]: I tested and it's not. I had it included at the start due to some issue, but I guess the cause was different.

File: projects/vdk-control-cli/cicd/build.sh (hunk 1 above)
Comment (tozka) [reply projects/vdk-control-cli/cicd/build.sh:33]: Please add this information in the comment or at least git description. We should be clear to future developer why we decided to make something if it's not clear from the code itself . 

Write in the ticket https://github.com/pytest-dev/pytest/issues/8539 that you are still seeing the issue in python v 3.10.xyz and in pytest vx.y ; Include snippet of logs and some details to help them root-cause.

Comment (gageorgiev) [reply projects/vdk-control-cli/cicd/build.sh]: I was accidentally using an older version of pytest due to it being pinned in requirements.txt. Removed the pin and the extra pytest install, hopefully it should work. Any idea why it was pinned originally?

Comment (tozka) [reply projects/vdk-control-cli/cicd/build.sh]: No idea. That's why it's important to put comments. It was copied over from the previous repo, so git history is gone to see details in the commit message (if there were any). Likely for no specific reason (copy pasted from somewhere else :) )

**Review by tozka — APPROVED** [review]

**Review by YanaZhivkova — APPROVED** [review]

**Pull request merged**